themes/confirmme/composer.json

```diff
@@ -2,7 +2,6 @@
   "name": "mautic/theme-confirmme",
   "description": "Mautic Confirmme Theme",
   "type": "mautic-theme",
-  "version": "1.0",
```

Comment:
This is no longer required, I believe - we are using tags for versioning which will be kept in sync with Mautic release versions.

Reply:
👍